New main script fixes

plusreed opened the pull request.

plusreed committed:
Fix getLogger bug

No quotation marks were around 'discord' which caused unintended behavior.

pep8speaks commented:
Hello @plusreed! Thanks for submitting the PR.

 - In the file [`main3.py`](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py), following are the PEP8 issues :

> [Line 3:16](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L3): [E401](https://duckduckgo.com/?q=pep8%20E401) multiple imports on one line
> [Line 36:80](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L36): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (98 > 79 characters)
> [Line 42:80](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L42): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (108 > 79 characters)
> [Line 44:80](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L44): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (100 > 79 characters)
> [Line 48:80](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L48): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (84 > 79 characters)
> [Line 75:80](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L75): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (95 > 79 characters)
> [Line 97:80](https://github.com/plusreed/foxpy/blob/aee0fccc522015858448ba90a2135754b1a09c8a/main3.py#L97): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (96 > 79 characters)

plusreed committed:
Fix load_extension error

Accidentally misspelled load_extension as load_extention, so no cogs were loaded

plusreed committed:
Adding CircleCI file

plusreed committed:
Adding test command

More just playing around at this point

plusreed committed:
Fixing possible bug

plusreed committed:
Update circle.yml

plusreed committed:
Update circle.yml

plusreed committed:
Update circle.yml
